pages/05.Campaigns/docs.en.md

```diff
@@ -7,25 +7,27 @@ taxonomy:
+Campaigns can be broadly categorized into three types:
 
-### Contact Driven Campaigns
+### Time driven campaigns
 
-A second type of campaign is the contact driven campaign. These are campaigns which trigger events based on interactions from the contact. These can occur as a result of the contact landing on a certain page, opening an email, spending a specified amount of time on a website, or any number of other activities. The response to these actions would be an email sent either immediately or at some point in the future.
+Time driven campaigns are the type of campaigns that center around specific timed events. These events can be anything but are usually in the form of emails. For instance, you can choose to trigger such email events after a delay of a predefined number of days or on a specific date in the future.
 
-### Mixed Campaigns
+### Contact driven campaigns
 
-Obviously you are not limited to creating either one kind of campaign or another separately. Mautic allows you to create campaigns which consist of both time driven items as well as contact driven actions. This powerful mixed campaign means actions will be driven by both specific dates or after specific timeframes as well as actions taken by a contact directly (as mentioned above).
+Contact driven campaigns are used to trigger events based on interactions with specific contacts. These interactions can occur as a result of the contact landing on a certain page, opening an email, spending a specified amount of time on a website, or any other activities. Such campaigns can be set up to respond to these actions typically by sending an email to the contact either immediately or at a specific time in future.
 
-### Campaign Actions
+### Mixed campaigns
 
-Email actions have been mentioned specifically as a particular use case, however there are many other responses the system can take. Other actions can include automatic assignment to a new segment, assigning a new point value, or an integration into a CRM or other system.
+Mautic allows you to create campaigns which consist a mix of both time driven campaigns as well as contact driven actions. This powerful mixed campaign strategy means that actions will be driven by both specific dates or after specific timeframes as well as actions taken by a contact directly.
+<br>
 
-### Campaign Automation
-
-One of the main benefits of this campaign workflow process is the ability to predefine these workflows and have them respond automatically to your contacts and timelines. This automation minimizes the amount of time required for manual contact activity and improves reliability of contact nurturing through consistent contact.
+> **Note**: Campaigns can trigger a variety of actions. Email actions are mentioned in the above campaign types only as an example. The other actions that a campaign can trigger include automatic assignment to a new segment, assigning a new point value, or an integration into a CRM or other systems.
```

Comment:
```suggestion
>>> Campaigns can trigger a variety of actions. Email actions are mentioned in the above Campaign types only as an example. The other actions that a Campaign can trigger include automatic assignment to a new Segment, assigning a new Point value, or pushing into an integration into a CRM or other systems.
```

Maybe consider linking this to the docs on Actions?

Reply:
About linking this to docs on Actions, it will be breaking the flow considering that we introduce the concept of Actions, Decisions, and Conditions only in the next chapter.